include/zephyr/drivers/adc/adc_voltage_divider.h

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
nit: `s/adc_voltage_divider/voltage_divider/` (file name)

Reply:
Hi, for your other comments, I've made the other changes as suggested. 

I'm not sure what you mean with this comment, because later down you say to keep the drivers under `adc/`. Also I don't understand the advantage of putting each driver 2 directories deep just to hold the one file?

Reply:
I mean to have `adc/voltage_divider.h` and not `adc/adc_voltage_divider.h` (ie double `adc`)

Reply:
The driver names have been modified to not duplicate `adc`.